lib/macho.rb

```diff
@@ -56,6 +56,6 @@ def self.codesign!(filename)
            "--preserve-metadata=entitlements,requirements,flags,runtime",
```

Comment:
Can this use `quiet_system` or some equivalent that doesn't show the output to users?

Reply:
This might be tough in just ruby-macho, since it doesn't have access to `quiet_system` or the other Homebrew helpers. We could use `Open3` or another stdlib module to control the output streams here, though.

Reply:
> We could use `Open3` or another stdlib module to control the output streams here, though.

👍🏻